website/default.nix

```diff
@@ -0,0 +1,115 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.nix-bitcoin-org.website;
+
+  nginxAddress = if config.nix-bitcoin.netns-isolation.enable then
+    config.nix-bitcoin.netns-isolation.netns.nginx.address
+  else
+    "localhost";
+
+  serviceAddress = service:
+    with config.services.${service};
+    "${address}:${toString port}";
+
+in {
+  options.nix-bitcoin-org.website = {
+    enable = mkEnableOption "nix-bitcoin.org website";
+  };
+
+  config = mkIf cfg.enable (mkMerge [
+  {
+    systemd.tmpfiles.rules = [
+      # Create symlink to static website content
+      "L+ /var/www - - - - ${./static}"
+    ];
+
+    networking.firewall.allowedTCPPorts = [
+      80
+      443
+    ];
+
+    security.acme = {
+      email = "nixbitcoin@i2pmail.org";
+      acceptTerms = true;
+    };
+
+    services.nginx = let
+      hostConfig = {
+        root = "/var/www";
+        locations."~ \"/btcpayserver/(apps|main|vendor|i|bundles|checkout|locales|)\"" = {
```

Comment:
Suggestion (The Github suggestion feature is unavailable for me right now): 
```
locations."~ /btcpayserver/(apps|main|vendor|i|bundles|checkout|locales)/"
```
The previous regexp had no limiting effect because it matched any URL with prefix `/btcpayserver/`:
`(...|)` matched the empty string and the final `/` was missing.

How did you find all the public sub-paths `apps|main|vendor|i|bundles|checkout|locales`?

Reply:
> How did you find all the public sub-paths `apps|main|vendor|i|bundles|checkout|locales`?

By going through the checkout process and watching `journalctl -fu nginx`. All these came up as errors because nginx was looking for them in `/var/www`.